src/Components/Aspire.Azure.Messaging.WebPubSub/AssemblyInfo.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire;
+using Aspire.Azure.Messaging.WebPubSub;
+using Azure.Messaging.WebPubSub;
+
+[assembly: ConfigurationSchema("Aspire:Azure:Messaging:WebPubSub", typeof(AzureMessagingWebPubSubSettings))]
+[assembly: ConfigurationSchema("Aspire:Azure:Messaging:WebPubSub:ClientOptions", typeof(WebPubSubServiceClientOptions))]
```

Comment:
```suggestion
[assembly: ConfigurationSchema("Aspire:Azure:Messaging:WebPubSub:ClientOptions", typeof(WebPubSubServiceClientOptions), exclusionPaths: ["Default"])]
```

We don't want to let the `Default` static property be generated in the json schema.